src/coreclr/nativeaot/Runtime/wasm/wasi_mmap.cpp

```diff
@@ -0,0 +1,27 @@
+#include <malloc.h>
+#include <string.h>
+
+#include "CommonTypes.h"
+#include "../CommonMacros.h"
```

Comment:
```suggestion
#include "CommonMacros.h"
```
I do believe this should work...

Reply:
Indeed, thanks